### DISCUSSION TITLE
KlipperScreen on Android

### DISCUSSION THREAD

**Discussion by Tiwatz** [xblax/flashforge_ad5m_klipper_mod#148] [Q&A]
I don't like the touchscreen on the printer so I tried to use an old Android phone as a screen following https://klipperscreen.readthedocs.io/en/latest/Android/
But no luck there for me since we don't have apt-get. 
I also tried the WIFI method but maybe this mod does not use `launch_KlipperScreen.sh` ?
It looks like in `KlipperScreen-start.sh` it should use it if it exists.

**Comment by Tiwatz** [comment]
I "solved" it by installing Klipper screen on my other printer and added the 5M Pro on that. Seems to work really good.

**Comment by xblax** [comment]
Just a comment for further reference:

The mod does not have apt-get but also no adb (from Android Debug Tools), therefore this kind of integration is not possible. If KlipperScreen is able to run on android on it's own with static configuration it should be doable. KlipperScreen could just connect to Moonraker on the printer.
